packages/client/src/index.ts

```diff
@@ -33,23 +33,20 @@ const _ = (async () => {
     key: 'com.twitter',
     universalResolverAddress: resolver,
   })
-  const avatar = await client.getEnsAvatar({
+  const avatar = await client.getEnsText({
     name: publicAddress,
+    key: 'avatar',
     universalResolverAddress: resolver,
   })
+
   const address = await client.getEnsAddress({
     name: publicAddress,
     universalResolverAddress: resolver,
   })
-  const name = await client.getEnsName({
-    address: '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266',
-    universalResolverAddress: resolver,
-  })
```

Comment:
Name is set in a contract called registrar. In the L2 context, this contract does not exist. So, this Name cannot be fetch.

Reply:
I think this contract lives on the L1 on [this address](https://github.com/ensdomains/ens-contracts/blob/staging/deployments/sepolia/ReverseRegistrar.json). I'm not sure we're gonna need this for Arbitrum, but any other approach will use it so you can't remove from client/index file since it is shared among other implementations